From 166ce57b57dd71e86bc70c7135dd8ef870dae126 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Fri, 17 Feb 2017 09:34:48 +0800
Subject: [PATCH] arm64: dts: rockchip: enable otg-port node of usb2-phy for
 rk3328-evb

This patch enable otg-port node of usb2-phy for dwc2 otg controller
on rk3328-evb board.

Change-Id: Ic6ce4beb2ba1814554e709a7d8af83a9ece9d7c9
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-evb.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
index 8c9e64c72e68..10a983c1ffdd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
@@ -118,11 +118,16 @@
 };
 
 &u2phy {
+	otg-vbus-gpios = <&gpio0 27 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
 	u2phy_host: host-port {
 		status = "okay";
 	};
+
+	u2phy_otg: otg-port {
+		status = "okay";
+	};
 };
 
 &u3phy {
-- 
2.35.3

